src/PackageVersions/Versions.php

```diff
@@ -3,27 +3,80 @@
 namespace PackageVersions;
 
 /**
- * This is a stub class: it is in place only for scenarios where PackageVersions
```

Comment:
This file should not have been committed

Reply:
Damn, my bad! That's also probably the cause behind the current CI failure...